internal/rbd/driver/driver.go

```diff
@@ -163,17 +177,6 @@ func (r *Driver) Run(conf *util.Config) {
 			"and replaced by CSI-Addons, see https://github.com/ceph/ceph-csi/issues/3314 for more details")
 		r.rs = NewReplicationServer(r.cs)
 	}
-	if !conf.IsControllerServer && !conf.IsNodeServer {
-		topology, err = util.GetTopologyFromDomainLabels(conf.DomainLabels, conf.NodeID, conf.DriverName)
-		if err != nil {
-			log.FatalLogMsg(err.Error())
-		}
-		r.ns, err = NewNodeServer(r.cd, conf.Vtype, topology)
-		if err != nil {
-			log.FatalLogMsg("failed to start node server, err %v\n", err)
-		}
-		r.cs = NewControllerServer(r.cd)
-	}
```

Comment:
This takes out the possibility of running cephcsi as both controller and provisioner. good to add this to release notes

Reply:
done added a note in tracker issue https://github.com/ceph/ceph-csi/issues/3336#issuecomment-1415731640 

opened todo issue for e2e and helm options.
https://github.com/ceph/ceph-csi/issues/3642